src/validateEmail.test.js

```diff
@@ -8,13 +8,57 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof validateEmail('test12@mail.com')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid email`, () => {
     expect(validateEmail('test838@gmail.com.'))
       .toBeTruthy();
   });
 
-  // write more tests here
+  it(`should return 'true' for email with '-' symbol`, () => {
+    expect(validateEmail('test-er@mail.com'))
+      .toBeTruthy();
+  });
+
+  it(`should return 'true' for email with '_' symbol`, () => {
+    expect(validateEmail('test_er@mail.com'))
+      .toBeTruthy();
+  });
+
+  it(`should return 'true' for email`
+  + `with letters, digits, hypens and dots`, () => {
+    expect(validateEmail('tes.t_er123@mail.com'))
+      .toBeTruthy();
+  });
```

Comment:
as per the test design techniques all these test will be in the one equivalence class
so please provide only one test which include all allowed chars